actor/kvcounter-tinygo/kvcounter_tinygo.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"strconv"
+
+	"github.com/wasmcloud/actor-tinygo"
+	httpserver "github.com/wasmcloud/interfaces/httpserver/tinygo"
+	keyvalue "github.com/wasmcloud/interfaces/keyvalue/tinygo"
+)
+
+func main() {
+	actor.RegisterHandlers(httpserver.HttpServerHandler(&KvcounterTinygo{}))
+}
+
+type KvcounterTinygo struct{}
+
+func (e *KvcounterTinygo) HandleRequest(ctx *actor.Context, req httpserver.HttpRequest) (*httpserver.HttpResponse, error) {
+	kv := keyvalue.NewProviderKeyValue()
```

Comment:
this should actually use the increment/atomic add operation rather than get, manual increment, and then set.

Reply:
Huzzah for simplicity! Good call